build_local.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
```

Comment:
we might add a check+warning concerning the need for a `GITHUB_API_TOKEN` env var, so someone doesnt get all the way through the process and then hit that issue once the npm scripts run